### DISCUSSION TITLE
[Question] Resizing page objects is proportional to the size of the window in real-time

### DISCUSSION THREAD

**Discussion by MrDIDJoy** [flet-dev/flet#4492] [Q&A]
### Question

I want to adaptive my PC app to different screen scales. I have already tried to implement something similar using [ResponsiveRow()](https://flet.dev/docs/controls/responsiverow), but I did not achieve the expected result: When resizing the window, only the lines changed, or rather the width of its objects, and the size of the text remained the same as it is. I want the values of ALL page objects and its sub-objects to change when the screen is changed, including the size of the text, and at the same time so that the size of the controls remains the same even after a strong change in the screen scale. Help please 🙏

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ap4499** [comment]
Hook up your logic to the following:
https://flet.dev/docs/controls/page/#on_resized

You might also want to have a look at the section of the blog where this changed in 0.24
https://flet.dev/blog/flet-v-0-24-release-announcement
go to section: Event-Handler subscription

If you pass the instance of your GUI objects, and then manipulate their attributes, that should work once you call object.update() or even page.update()
